src/olympia/hero/tests/test_models.py

```diff
@@ -1,32 +1,35 @@
 from django.core.exceptions import ValidationError
 
 from olympia.amo.tests import addon_factory, TestCase
-from olympia.constants.promoted import RECOMMENDED
-from olympia.hero.models import PrimaryHero, SecondaryHero, SecondaryHeroModule
-from olympia.promoted.models import PromotedAddon, PromotedApproval
+from olympia.amo.tests.test_helpers import get_uploaded_file
+from olympia.hero.models import (
+    PrimaryHero, PrimaryHeroImage, SecondaryHero, SecondaryHeroModule)
+from olympia.discovery.models import DiscoveryItem
 
 
 class TestPrimaryHero(TestCase):
+    def setUp(self):
+        uploaded_photo = get_uploaded_file('transparent.png')
+        self.phi = PrimaryHeroImage.objects.create(custom_image=uploaded_photo)
+
     def test_image_url(self):
         ph = PrimaryHero.objects.create(
-            promoted_addon=PromotedAddon.objects.create(addon=addon_factory()),
-            image='foo.png')
+            disco_addon=DiscoveryItem.objects.create(addon=addon_factory()),
+            select_image=self.phi)
```

Comment:
you don't want this change - PrimaryHero doesn't fk to DiscoveryItem any longer.  It's just the kwarg change from `image` that's needed